FlowCrypt/Models/Common/Keypair.swift

```diff
@@ -8,7 +8,7 @@
 
 import Foundation
 
-struct KeyInfo {
+struct Keypair {
```

Comment:
Let's change this one too for consistency

Reply:
very good